config/settings/development.yml

```diff
@@ -7,6 +7,7 @@ cis2:
 
 mesh:
   base_url: https://localhost:8700
+  disable_ssl_verification?: true
```

Comment:
It's a little bit weird that this has a ? in it since it could be an ENV var but I guess it works!

Reply:
Hrm, ya, a little weird. I'll remove it just in case.